libs/ledger-live-common/src/families/internet_computer/setup.ts

```diff
@@ -0,0 +1,52 @@
+// Goal of this file is to inject all necessary device/signer dependency to coin-modules
+
+import { createBridges } from "@ledgerhq/coin-internet_computer/bridge/index";
+import Transport from "@ledgerhq/hw-transport";
+import ICP from "@zondax/ledger-icp";
+import icpResolver from "@ledgerhq/coin-internet_computer/signer/index";
+import { signMessage } from "@ledgerhq/coin-internet_computer/hw-signMessage";
+import type { Account, Bridge } from "@ledgerhq/types-live";
+import makeCliTools from "@ledgerhq/coin-internet_computer/test/cli";
+import { CreateSigner, createResolver, executeWithSigner } from "../../bridge/setup";
+import { Resolver } from "../../hw/getAddress/types";
+import { TransactionStatus, Transaction } from "@ledgerhq/coin-internet_computer/types/index";
+import { ICPGetAddrResponse, ICPSignature, ICPSigner } from "./types";
+import { getPath, isError } from "./utils";
+
+const createSigner: CreateSigner<ICPSigner> = (transport: Transport) => {
+  const icp = new ICP(transport);
+  return {
+    showAddressAndPubKey: async (path: string): Promise<ICPGetAddrResponse> => {
+      const r = await icp.showAddressAndPubKey(getPath(path));
+      isError(r);
```

Comment:
`isError` is used only here, move the fonction here like in `filecoin` create a `common.ts`

Reply:
[`414ed5f` (#8560)](https://github.com/LedgerHQ/ledger-live/pull/8560/commits/414ed5f4f8fcfea3d789ff825a55c85045f2317b#diff-3eb81eecd857d372d1d69641bee437b2aa79a282bca5a5178c16c4accc91d3ae)